lgc/patch/NggPrimShader.cpp

```diff
@@ -1094,7 +1094,7 @@ void NggPrimShader::constructPrimShaderWithoutGs(Module *module) {
     m_builder->SetInsertPoint(writeVertCullDataBlock);
 
     // Write vertex position data
-    writePerThreadDataToLds(position, m_nggFactor.threadIdInSubgroup, LdsRegionVertPosData, true);
+    writePerThreadDataToLds(position, m_nggFactor.threadIdInSubgroup, LdsRegionVertPosData, 0, true);
```

Comment:
why the offset of vertex pos is always 0. if it is expected, could you add a comment for it? thanks!

Reply:
Originally, `writePerThreadDataToLds` had four parameters and the last one `useDs128` was a default parameter. Now, this function takes an additional default parameter `offsetInRegion` before `useDs1128` to address the data within a region. In most cases, `offsetInRegion` and `useDs128` needn't be set. `offsetInRegion` is used to support multi-stream because each stream has its own sub-region. And `useDs128` is for position data fetching/exporting of position data region. I will update function parameter comments to make it clear.